### ISSUE TITLE
Sampling bias (S dropout and others)

### ISSUE THREAD

**Issue by acx01b** [hodcroftlab/covariants#73]
Hi, there are some chances that this website will be analyzed by a lot of non-experts, so maybe add a disclaimer about identified biases: 

- S dropout targeted sequencing causing a huge increase in 69- variants in January (501Y, 439K, 484).

Optional:
- The large Y453F peak in Denmark this summer was (partly?) caused by sampling targeted on mink farms.
- Inside a country some regions are more sampled than others at different times and this is not always correlated to the number of cases in each region (as an exception [UK](https://www.cogconsortium.uk/wp-content/uploads/2021/01/COG-UK-geo-coverage_2021-01-18_summary.pdf) and [Denmark](https://www.covid19genomics.dk/statistics) are providing some data on their sampling method) 
- ?

The Europe biased analysis is mentioned in Home.md.

**Comment by emmahodcroft** [comment]
@acx01b Thank you for this suggestion! This is really helpful. I have modified the 'intro' to the 'Per Country' page to both discuss the bias caused by 'S-drop-out' preferential sequencing, and the fact that sequences may not be representative of the country as they may come from just one area. 

I've also added a bit to the S:Y453F page about possible bias due to sampling of mink farms in Denmark.
